composio/local_tools/local_workspace/cmd_manager/actions/clone_github.py

```diff
@@ -37,28 +46,38 @@ class GithubCloneCmd(BaseAction):
     _display_name = "Clone Github Repository Action"
     _request_schema = GithubCloneRequest
     _response_schema = GithubCloneResponse
+    command = (
```

Comment:
What is the point of this line?

Reply:
not needed --> self.command is needed to add in logs. Have added that later in method forgot to remove that one